test/e2e/pipeline/pipeline_test.go

```diff
@@ -192,6 +192,17 @@ Waiting for logs to be available...
 
 	time.Sleep(1 * time.Second)
 
+	t.Run("Get list of Pipelineruns using field-selector", func(t *testing.T) {
```

Comment:
Looks great! Can you just add something for the pipelinerun test to assert the output contains only the filtered pipelinerun? 

This may need to include creating a second pipelinerun?

Reply:
> Looks great! Can you just add something for the pipelinerun test to assert the output contains only the filtered pipelinerun?
>
I was trying to do this but not sure how to assert with `pr`. Do I need to use template?
for taskruns there are already func defined so do I need to write something like [ListAllTaskRunsOutput](https://github.com/tektoncd/cli/blob/6ff0d05bf0df01ff221bc39ce111023bcad7560e/test/e2e/build_templates.go#L332) ? or is there any other way?

Reply:
Maybe you could do something like:

```
assert.Check(t, strings.Contains(res.Stdout, pipelineRunGeneratedName), "String does not contain expected PipelineRun name")
```